nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/security/PermissionAutomationTest.java

```diff
@@ -87,14 +101,26 @@ public void initRepo() throws Exception {
 
         when(administratorGroupsProvider.getAdministratorsGroups()).thenReturn(
                 Collections.singletonList("administrators"));
+
+        DocumentModel testUser = userManager.getBareUserModel();
```

Comment:
You could have mocked the `userManager.getUserModel` method to avoid requiring the `PlatformFeature` creating and deleting a user.